airflow/config_templates/default_airflow.cfg

```diff
@@ -122,6 +122,12 @@ security =
 # values at runtime)
 unit_test_mode = False
 
+# Logging configuration path
+logging_config_path = airflow.logging.airflow_logging_config.AIRFLOW_LOGGING_CONFIG
+
+# Name of handler to read task instance logs
+task_log_reader = airflow.task
```

Comment:
In `views.py` it checks whether`handler.name` equals to `airflow.task`. Here I am assuming `handlers.name` returns the name of the handler defined in config not the full module name (tested log rendering locally and it works)